personalized_nlp/experiments/goemo_single_ann_tune.py

```diff
@@ -0,0 +1,133 @@
+import os
+import torch
+import random
+import numpy as np
+from itertools import product
+
+from personalized_nlp.learning.train import train_test
+from personalized_nlp.models import models as models_dict
+from personalized_nlp.settings import LOGS_DIR
+from pytorch_lightning import loggers as pl_loggers
+from personalized_nlp.datasets.goemotions.go_emotions import GoEmotionsDataModule
+
+from personalized_nlp.utils.callbacks.optimizer import SetWeightDecay
+from personalized_nlp.utils.callbacks.transformer_lr_scheduler import TransformerLrScheduler
+
+def seed_everything():
+    torch.manual_seed(0)
+    random.seed(0)
+    np.random.seed(0)
+
+torch.cuda.empty_cache()
+
+os.environ["CUDA_VISIBLE_DEVICES"] = '0'
+os.environ["WANDB_START_METHOD"] = "thread"
+
+if __name__ == "__main__":
+    regression = True
+    datamodule_cls = GoEmotionsDataModule
+    embedding_types = ['roberta']
+    model_types = ['baseline']
+    wandb_project_name = 'goemotions_baseline_single_ann_tune'
+    fold_nums = 10
+    min_annotations_per_text = 2
+
+    min_word_counts = [50]
+    words_per_texts = [128]
+
+    batch_size = 16
+    dp_embs = [0.25]
+    embedding_dims = [50]
+    epochs = 20
+    lr_rates = [1e-5, 3e-5, 5e-5]
+    weight_decay = 0.01
+    nr_frozen_epochs = 5
```

Comment:
I presume it should be 0 to perform finetuning from the beginning of the training.